dropshot/src/handler.rs

```diff
@@ -139,7 +141,7 @@ impl RequestInfo {
     ///
     /// This is provided for source compatibility.  In previous versions of
     /// Dropshot, `RequestContext.request` was an
-    /// `Arc<Mutex<hyper::Request<hyper::Body>>>`.  Now, it's just
+    /// `Arc<Mutex<hyper::Request<impl Body>>>`.  Now, it's just
```

Comment:
I'm not sure how to handle this. It might not be relevant any more in terms of migration. I'd like @davepacheco to look at this PR generally, but this in particular.

Reply:
I think that was just a string replacement that never got switched back to `dropshot::Body` since the compiler didn't notice. Reading the full docs, seems like this method is kept around for compatibility. It's saying "this is what the types used to be, but it's not anymore." :shrug: